pallet-chainlink-feed/src/lib.rs

```diff
@@ -489,12 +491,6 @@ pub mod pallet {
 		NotPendingPalletAdmin,
 		/// Round zero is not allowed to be pruned.
 		CannotPruneRoundZero,
-		/// The given pruning bounds don't cause any pruning with the current state.
-		NothingToPrune,
-		/// There is no valid data, yet, so the conditions for pruning are not met.
-		NoValidRoundYet,
-		/// The pruning should leave the rounds story in a contiguous state (no gaps).
-		PruneContiguously,
```

Comment:
SInce we now prune automatically, pruning is targets valid rounds only, and occurs only when new rounds are requested.

`ensure!(pruning_window > 0, Error::<T>::CannotPruneRoundZero);` ensures we have a valid window and

https://github.com/ChainSafe/chainlink-polkadot/pull/13/files#diff-987a170684786ff2b29a8acc32e56cacfe49e9158322b26e36cc67ec6a20c226R563 ensures we start at a valid round.

So `NoValidRoundYet` won't happen on an erroneous user input.
 `PruneContiguously` was introduced because the admin manually could request rounds to prune. Since we now prune always the oldest value, this isn't an issue anymore.